moveit_setup_assistant/src/collisions_updater.cpp

```diff
@@ -36,6 +36,7 @@
 
 #include <moveit/setup_assistant/tools/moveit_config_data.h>
 #include <moveit/rdf_loader/rdf_loader.h>
+#include <rviz_common/logging.hpp>
```

Comment:
I see now why you are depending on rviz, please don't make this depend on rviz for logging.  Please use standard rclcpp logging.  Here is an example of how to do that: https://github.com/ros-planning/moveit2/blob/26a2ab7a4a45339881416a25e4af700027de184d/moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp#L65

One thing about that example.  It defines a static global and I'll allow that as it is all over the codebase.  If you'd like to do it in a nicer way though please don't create a static global in the global namespace, instead create a normal static in an anonymous namespace for the LOGGER object.